x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/columns.tsx

```diff
@@ -191,12 +201,14 @@ export const getColumns = (
       'data-test-subj': 'mlAnalyticsTableRowDetailsToggle',
     },
     {
-      field: DataFrameAnalyticsListColumn.id,
+      // field: DataFrameAnalyticsListColumn.id,
```

Comment:
Is this commented out needed?

Reply:
I'll fix that. Thanks.

Reply:
This is now cleaned up in https://github.com/elastic/kibana/pull/65778/commits/1ec4fde438331af6d9bdb62f1ffe80563ca3e110